Corona-Warn-App/src/main/java/de/rki/coronawarnapp/tracing/ui/details/items/survey/UserSurveyBox.kt

```diff
@@ -0,0 +1,41 @@
+package de.rki.coronawarnapp.tracing.ui.details.items.survey
+
+import android.view.ViewGroup
+import androidx.annotation.LayoutRes
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.databinding.TracingDetailsAccessSurveyCardBinding
+import de.rki.coronawarnapp.datadonation.survey.Surveys
+import de.rki.coronawarnapp.tracing.ui.details.TracingDetailsAdapter
+import de.rki.coronawarnapp.tracing.ui.details.items.DetailsItem
+
+class UserSurveyBox(
```

Comment:
Call it `Card` instead of `Box` for consistent naming?

Reply:
thought first the same. 
But there are many cases where the "box" is used.
would like to let it box if nothing speaks against it
![image](https://user-images.githubusercontent.com/46747780/106754136-e4506400-662c-11eb-8fb2-5d7f8e691951.png)

Reply:
Ok, then its fine 😉